### PR TITLE
Support for `azurerm_api_management_notification_recipient_email`

### PR DESCRIPTION
**Pull request by yupwei68**
Partial Fix https://github.com/terraform-providers/terraform-provider-azurerm/issues/12848

--- PASS: TestAccApiManagementNotificationRecipientEmail_basic (4169.09s)
--- PASS: TestAccApiManagementNotificationRecipientEmail_requiresImport (4008.43s)

### CODE REVIEW

File: azurerm/internal/services/apimanagement/api_management_notification_recipient_email.go [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+func (r ApiManagementNotificationRecipientEmailResource) Arguments() map[string]*pluginsdk.Schema {
+	return map[string]*pluginsdk.Schema{
+		"api_management_id": {
+			Type:         pluginsdk.TypeString,
+			Required:     true,
+			ForceNew:     true,
+			ValidateFunc: validate.ApiManagementID,
+		},
+
+		"notification_name": {
```
Comment (katbyte) [inline azurerm/internal/services/apimanagement/api_management_notification_recipient_email.go:37]: should this be
```suggestion
		"notification_type": {
```

**Review by katbyte — CHANGES_REQUESTED** [review]
Thanks @yupwei68 - just one comment

**Comment by yupwei68** [comment]
@kt Tests have passed. Please continue reviewing.

**Review by katbyte — APPROVED** [review]
Thanks @yupwei68 - LGTM 🏗️

**Pull request merged**